docs/sources/tempo/metrics-generator/span_metrics.md

```diff
@@ -150,6 +153,39 @@ The following intrinsic kinds are available for filtering.
 - `SPAN_KIND_PRODUCER`
 - `SPAN_KIND_CONSUMER`
 
+
+#### Filter policies
+
+Each filter policy is evaluated in order for every span to determine if the
+span should be included for metrics.  Both `include` and `exclude` policies are
+used to *reject* spans from the metrics.  With no filter polices applied, all
+spans are included.
```

Comment:
I'm having some trouble parsing the first sentence. Do you mean 1) that every filter policy is evaluated in a particular order (i.e., policy A first, policy B, policy C, etc.) or 2) that each policy is used to evaluated every span to determine whether a span should be included for metrics? 
```suggestion
Each filter policy is evaluated in order for every span to determine if the
span should be included for metrics.  Both `include` and `exclude` policies are
used to *reject* spans from the metrics.  With no filter polices applied, all
spans are included.
```

Reply:
Okay, reading later down I think you're referring to the order of the filter policies.

Reply:
Okay I've updated.  The policies are evaluated in order yes (0,1,2), but only until there is a match to include without a matching exclude on the same policy.  Hows that read?